C-Plus-Plus/adjacent.cpp

```diff
@@ -0,0 +1,36 @@
+/* The adjacent_difference algo produces a new sequence
+ in which each element is difference between adjacent 
+ elements in original sequence. */
+
+#include <iostream>
+#include <vector>
+#include <numeric>
+using namespace std;
+
+int main(){
```

Comment:
Don't put up the entire code logic in `main()` function. Use `main()` only for the driver code

Reply:
ok

Reply:
Bhaiya I have tried but i am not able to make function because it is a basic example of adjacent difference. if Possible may u merge this or text time I will surely keep in mind.